Airports/EGMC/SMR/Regions.txt

```diff
@@ -1,13 +1,13 @@
 ;Blue_Background
 REGIONNAME Southend
-blueBackground N051.31.48.184 E000.45.48.948
```

Comment:
Hmm, I'm trying to remember if these background colours were defined separately for a reason... I shall have a think!

Reply:
That is probably my fault - I’ll have a look in the morning

Reply:
Well both of the colour designators exist in Colors.txt, but they're both the same colour! So just trying to think if there is a reason for this... (@KHardern ?)

Reply:
Internet's been out all morning (thanks BT) but I've changed `smrBlue` and `smrBlack` to `blueBackground` and `blackBackground` respectively - it future-proofs the changes so we can change `smrBlue` and `smrBlack` without affecting the backgrounds of SMR displays.